AWS/legos/aws_list_expiring_access_keys/aws_list_expiring_access_keys.json

```diff
@@ -11,6 +11,6 @@
     "action_nouns": ["expiring","access","aws"],
     "action_is_check": true,
     "action_categories": [ "CATEGORY_TYPE_IAM", "CATEGORY_TYPE_SECOPS"],
-    "action_next_hop": "<UUID of lego for remediation if check fails>",
+    "action_next_hop": ["e591113f7afc699ee564d67ef912ea2d689acc91d7640a2a05e68c039153bd33","0297f6c80f0416d10484fa2593510515eef2900add97924e3e73beaab5fea819","4ce21d2ac0824cafdddbb4245ffcb1d4c34786ed68c075fb1041eb8c7e22f01d"],
     "action_next_hop_parameter_mapping": {}
```

Comment:
The action_next_hop_parameter_mapping can be made like:

{
  "title": "Test",
  "type": "object",
  "properties": {
    "result": {
      "title": "Result",
      "description": "Result of the Execution",
      "type": "boolean"
    },
    "objects": {
      "title": "List of Objects",
      "description": "List of Objects after exection",
      "type": "array",
      "items": {}
    }
  },
  "required": [
    "result",
    "objects"
  ]
}